lang/de/miquiz.php

```diff
@@ -5,31 +5,36 @@
 $string['modulename'] = 'MI-Quiz';
 $string['modulenameplural'] = 'MI-Quizze';
 
-//$instance_name ='';
```

Comment:
We had this, because when we install the plugin the first time, the get_config call (next line) throws an error... So the workaround for a new install is to manually set the instance_name.